.github/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security policy
+
+Mozilla takes the security of our software seriously. If you believe you have found a security vulnerability in PDF.js, please report it to us as described below.
+
+## Reporting security vulnerabilities
+
+**Please don't report security vulnerabilities through public GitHub issues.**
+
+Instead, please report security vulnerabilities in [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=PDF%20Viewer&groups=firefox-core-security), making sure that the checkbox in the "Security" section is checked so the required access controls are automatically configured. The Mozilla security team will process the bug as described in [Mozilla's security bugs policy](https://www.mozilla.org/en-US/about/governance/policies/security-group/bugs).
```

Comment:
Maybe we could add a screenshot
![image](https://github.com/mozilla/pdf.js/assets/5641725/552ce7d6-8a84-4650-bba7-61781a98abdc)
to help the user to see what we're talking about here. I know some people are sometimes a bit scared by the Bugzilla ui.

Reply:
Good idea; I have included this in the new commit.